hadoop-hdds/client/src/main/java/org/apache/hadoop/hdds/scm/XceiverClientGrpc.java

```diff
@@ -107,6 +111,7 @@ public XceiverClientGrpc(Pipeline pipeline, Configuration config,
         OzoneConfigKeys.OZONE_NETWORK_TOPOLOGY_AWARE_READ_KEY,
         OzoneConfigKeys.OZONE_NETWORK_TOPOLOGY_AWARE_READ_DEFAULT);
     this.caCert = caCert;
+    this.getBlockDNcache = new HashMap<>();
```

Comment:
XceiverClientGrpc is shared across threads, do we need to make this ConcurrentHashMap?

Reply:
yes, this should be concurrent.

Reply:
Thanks @bharatviswa504.
Fixed it.